sample-node-config/aws/docker/docker-compose.yaml

```diff
@@ -0,0 +1,100 @@
+version: "3.9"
+
+services:
+  cardano-node:
+    image: inputoutput/cardano-node:latest
+    restart: always
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "100m"
+        max-file: "5"
+    volumes:
+      - .:/data
+      - ./devnet:/devnet
+    working_dir: /data
+    environment:
+      - CARDANO_SOCKET_PATH=/devnet/node.socket # used by cardano-node
+      - CARDANO_NODE_SOCKET_PATH=/devnet/node.socket # used by cardano-cli
+    command:
+      [ "run"
+      , "--config", "/devnet/cardano-node/config.json"
+      , "--topology", "/devnet/cardano-node/topology.json"
+      , "--database-path", "/devnet/db"
+      ]
+
+  hydra-node:
+    image: abailly/hydra-node:latest
+    restart: always
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "100m"
+        max-file: "5"
+    volumes:
+      - .:/data:ro
+      - ./devnet:/devnet
+    ports:
+      - "5001:5001"
+    command:
+      [ "--node-id", "1"
+      , "--api-host", "0.0.0.0"
+      , "--host", "0.0.0.0"
+      , "--monitoring-port", "6000"
+      , "--hydra-scripts-tx-id", "${HYDRA_SCRIPTS_TX_ID}"
+      , "--hydra-signing-key", "/data/hydra-key.sk"
+      , "--cardano-signing-key", "/data/cardano-key.sk"
+      , "--ledger-genesis", "/data/genesis-shelley.json"
+      , "--ledger-protocol-parameters", "/data/protocol-parameters.json"
+      , "--network-id", "${NETWORK_MAGIC}"
+      , "--node-socket", "/devnet/node.socket"
+      # peers
+      , "--peer", "35.233.17.169:5001"
+      , "--hydra-verification-key", "/data/arnaud-hydra.vk"
+      , "--cardano-verification-key", "/data/arnaud-cardano.vk"
```

Comment:
@ffakenz were you using the same credential names as arnaud? :)

Reply:
these are the peer's VKs (eg. my machine's). It's not worse to have `arnaud` here than to have the hardcode IP address :)

Reply:
Ah yes.. of course